api/version/2.8.0.md

```diff
@@ -1,6 +1,6 @@
-# ECE 2.7.0-ms47 Command Declaration
+# ECE 2.8.0 Command Declaration
```

Comment:
We should keep the `2.7.0-ms47.md` file intact and add `2.8.0.md` as a new one

Reply:
@karencfv I'm confused here. The file is named `2.8.0.md` so does the header inside it. I don't understand what I'm supposed to do here 😕

Reply:
I can see why my comment would be confusing since I cannot comment on the actual file name but rather on a random line 😄 

What I see on the GH UI is this:
![Screen Shot 2020-11-11 at 10 45 26 AM](https://user-images.githubusercontent.com/7670244/98737518-2e012800-240b-11eb-85bc-2daebcbde84a.png)

So it looks like you are renaming the existing `2.7.0-ms47.md` file to `2.8.0.md` :)